app/libs/resizable-columns.js

```diff
@@ -221,8 +221,8 @@ export default class {
       let diff = this.tableWidth - totalWidth;
       while (diff > 0) {
         columns
-          .filter((col) => col.visible)
-          .sortBy('width')
+          .filter((x) => !!x.visible)
```

Comment:
Was there a reason this was changed from `col` to `x`? `x` does not indicate what the item is at all.

Reply:
Probably bad rebase